src/pymmcore_widgets/_mda/_util/_labeled_slider.py

```diff
@@ -1,5 +1,11 @@
+from __future__ import annotations
+
+from typing import Any
+
 from qtpy import QtCore, QtGui, QtWidgets
 
+FIXED = QtWidgets.QSizePolicy.Policy.Fixed
```

Comment:
note, Qt6 is removing globally scoped enums, so `QSizePolicy.Fixed` will need to become `QSizePolicy.Policy.Fixed`, and `Qt.Horizontal` will need to be `Qt.Orientation.Horizontal`, etc... it's quite verbose, I know, so sometimes I'll make a tmp variable like this. (but you can also use the full enum inline)